mxpy-up.py

```diff
@@ -321,13 +298,154 @@ def run_post_install_checks():
+
+The "mxpy" command shortcut will not be available until you add the the directory "{sdk_path}" to the system PATH.
+###############################################################################
+Do you understand the above?
+""")
+        confirm_continuation(interactive)
+        return
+
+    profile_files_formatted = "\n".join(f" - {file}" for file in profile_files)
+    profile_files_contents = [profile_file.read_text() for profile_file in profile_files]
+    any_old_export_directive = any(old_export_directive in content for content in profile_files_contents)
+    any_new_export_directive = any(new_export_directive in content for content in profile_files_contents)
+
+    if any_old_export_directive:
+        # We don't perform the removal automatically (a bit risky)
+        print(f"""
+###############################################################################
+It seems that the old path "~/elrondsdk" is still configured in shell profile.
+
+Please MANUALLY remove it from the shell profile (now or after the installer script ends).
+
+Your shell profile files:
+{profile_files_formatted}
+
+The entry (entries) to remove: 
+    {old_export_directive}
+###############################################################################
+Make sure you understand the above before proceeding further.
+###############################################################################
+""")
+        confirm_continuation(interactive)
+
+    if any_new_export_directive:
+        # Note: in some (rare) cases, here we'll have false positives (e.g. if the export directive is commented out).
+        print(f"""
+###############################################################################
+It seems that the path "~/multiversx-sdk" is already configured in shell profile.
+
+To confirm this fact, check the shell profile (now or after the installer script ends). 
+
+Your shell profile files:
+{profile_files_formatted}
+
+The entry to check (it should be present): 
+    {new_export_directive}.
+###############################################################################
+Make sure you understand the above before proceeding further.
+###############################################################################
+""")
+        confirm_continuation(interactive)
+        return
+
+    choice = input(f"""
+###############################################################################
+In order to use the "mxpy" command shortcut, you have the choice to either:
+ - manually extend the PATH variable to include "~/multiversx-sdk"
+ - allow this installer script to do it for you.
+
+In order to manually extend the PATH variable, add the following line to your shell profile file upon installation:
+
+    export PATH="${{HOME}}/multiversx-sdk:${{PATH}}"
+
+Your shell profile files:
+{profile_files_formatted}
+
+Upon editing the shell profile file, you may have to RESTART THE USER SESSION for the changes to take effect.
+
+Please pick an option:
```

Comment:
I don't know if it is clear that we need to input "a" or "b" in order to pick an option.

Reply:
Fixed, added an extra line.